### PR TITLE
Reza react week1

### PR DESCRIPTION
**Pull request by AliRezaHassanloo2731**
main

### CODE REVIEW

File: app/about_us/OurCrew/OurCrew.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+export default function OurCrew() {
+  // TASK - React 1 week 1
+  // Create the "Our Crew section"
+  // Use the descriptions provided in /app/about_us/README.md
+  // Use the pictures from /public/crew
+  // Some inspiration ideas found in /data/inspiration_about_us
+  return (
+    <>
+      <div className={styles.grid}>
```
Comment (ddobby94) [inline app/about_us/OurCrew/OurCrew.js:11]: We can just use the `<div>` as the root html node here:
```
return (
  <div>
    ...
  </div>
)
```

So you can get rid of the empty brackets `<>`

File: app/about_us/ourValue/OurValues.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          We believe in the power of collective exploration. Our journeys are
+          not just about reaching new destinations; they are about building a
+          community of space enthusiasts who share a passion for the stars.
+        </p>
+      </div>
+    </div>
+  );
+}
+
+function Value() {
```
Comment (ddobby94) [inline app/about_us/ourValue/OurValues.js:56]: I don't think this Component is being used anywhere, you can delete it.

File: app/about_us/page.js [hunk 3]
```diff
@@ -45,18 +13,16 @@ export const Crew = () => {
...
         </section>
         <section className="card">
           <h2>The crew</h2>
-          <OurCrew/>
+          <OurCrew />
         </section>
-
-         {/* TASK - React 1 week 1 */}
-         {/* Add in the "OurPartners" component here */}
+        <OurPartners />
```
Comment (ddobby94) [inline app/about_us/page.js:22]: It seems like you are not following the same pattern for `<OurPartners />` as you are doing for the Our Values and Our Crew. It is best to stay consistent with these:
```
 <section className="card">
    <h2>The crew</h2>
     <OurCrew />
  </section>
   <section className="card">
    <h2>Our Partners</h2>
     <OurPartners />
  </section>
```

File: app/about_us/ourValue/OurValues.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+// TASK - React 1 week 1
+// After you are finished with creating the page, move the OurValues, OurCrew, OurPartners components into their own files
+// OurValues.js, OurCrew.js, OurPartners.js should live in this folder
+// import and use the components from the newly created files
+export default function OurValues() {
+  // TASK - React 1 week 1
+  // Create the "Our Values" section
+  // Use the descriptions provided in /app/about_us/README.md
+  // Some inspiration ideas found in /data/inspiration_about_us
```
Comment (ddobby94) [inline app/about_us/ourValue/OurValues.js:11]: These comments can be deleted in the upcoming HWs :)

File: components/ui/Footer.js [hunk 5]
```diff
@@ -41,6 +44,11 @@ export const Footer = () => {
           <li>
             <a href="https://google.com">On the streets at night</a>
           </li>
+          <li>
+            <a href="https://www.linkedin.com/in/alireza-hassanloo-415356239/">
+              Linkdin
```
Comment (ddobby94) [inline components/ui/Footer.js:49]: There is a small typo here 😄

File: app/about_us/OurCrew/OurCrew.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+            Anousheh Ansari (Persian: انوشه انصاری; née Raissyan;[2] born
+            September 12, 1966) is an Iranian American engineer and co-founder
+            and chairwoman of Prodea Systems
+          </p>
+        </div>
+        <div>
+          <img
+            className={styles.image}
+            src="./crew/image-douglas-hurley.png"
+            alt="anushe"
```
Comment (ddobby94) [inline app/about_us/OurCrew/OurCrew.js:28]: This `alt="..."` text is misleading for non-sighted users, so you can either change it or remove it.

- More info on alt tag: https://www.w3.org/WAI/tutorials/images/
- Videos about Accessibility testing, and how people use assistive technologies:
  - Short intro video - https://www.youtube.com/watch?v=HE2R86EZPMA
  - Longer and detailed video from a non-sighted user - https://www.youtube.com/watch?v=OUDV1gqs9GA

File: app/about_us/OurPrtner/OurPartners.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+        <img src="business_partners/alphabet-logo.png" alt="alphabet-logo" />
+        <img src="business_partners/amazon_logo.png" alt="amazon_logo" />
+        <img src="business_partners/CBC_Logo_White.png" alt="CBC_Logo_White" />
+        <img
+          src="business_partners/Microsoft-Logo-white.png"
+          alt="Microsoft-Logo-white"
+        />
+        <img
+          src="business_partners/Microsoft-Logo-white.png"
+          alt="Microsoft-Logo-white"
```
Comment (ddobby94) [inline app/about_us/OurPrtner/OurPartners.js:27]: Same comment about the `alt="..."` text

**Review by ddobby94 — COMMENTED** [review]
Nice job!

Most important thing is that you understand how React components are created and it seems like you are handling them super well! 😄

File: app/about_us/ourValue/OurValues.js (hunk 2 above)
Comment (AliRezaHassanloo2731) [reply app/about_us/ourValue/OurValues.js:56]: ok

File: app/about_us/OurCrew/OurCrew.js (hunk 1 above)
Comment (AliRezaHassanloo2731) [reply app/about_us/OurCrew/OurCrew.js:11]: I think it was for something, but when i delete it nothing hapepened, so i delete it

File: app/about_us/page.js (hunk 3 above)
Comment (AliRezaHassanloo2731) [reply app/about_us/page.js:22]: Oh, thanks

File: app/about_us/ourValue/OurValues.js (hunk 4 above)
Comment (AliRezaHassanloo2731) [reply app/about_us/ourValue/OurValues.js:11]: in the week 3 i'll delete all comments

**Pull request merged**